send/umbraco_mu

```diff
@@ -18,6 +18,7 @@ send_lib.check_destination_format(destination, destination_type)
 
 auth = send_lib.get_auth_credentials(SERVICE_NAME, destination)
 
+send_lib.create_lock(SERVICE_NAME, destination)
```

Comment:
maybe this should be a separate commit (or at least a mention in the message)?

Reply:
added to commie message